### ISSUE TITLE
Too Many Clients Error when running a Job

### ISSUE THREAD

**Issue by jeffkala** [nautobot/nautobot-plugin-nornir#50]
### Environment
* Python version:  3.7+
* Nautobot version:  1.2.x
* nautobot-plugin-nornir version:  current + develop branch

### Steps to Reproduce
1. Setup golden config plugin
2. Run against high number of devices.

Currently an intermittent issue that has only been hit a few times without exact replication steps.

### Expected Behavior
Job completes as expected.

### Observed Behavior
Job succeeds and then every device thereafter fails with `too many clients` error in the job result.

### Initial Theories
the problem is a conflict between uwsgi and celery spawning processes and leaving connections open
basically by setting to serial, we just leave it up to uwsgi to thread it and the connections aren’t left open

This is a symptom not the issue.  Needs to be investigated further.

### Workaround
Set the runner to serial and disabled the num of workers.

**Comment by Copter64** [comment]
I ended up lowering the number of workers from 20 to 10.  I'll keep an eye out if it occurs again. But so far I've run the GC job several times without the too many clients errors.

**Comment by itdependsnetworks** [comment]
Fixed in https://github.com/nautobot/nautobot-plugin-golden-config/pull/352
